pkg/utils.py

```diff
@@ -299,37 +321,46 @@ def json_to_junit(test_name, data_json, metrics_config):
         testsuites, "testsuite", name=f"{test_name} nightly compare"
     )
     failures_count = 0
-    test_count=0
-    for run in data_json:
-        run_data = {
-            str(key): str(value).lower()
-            for key, value in run.items()
-            if key in ["timestamp"]
-        }
-        for metric, value in run["metrics"].items():
-            test_count+=1
-            failure = "false"
-            if not value["percentage_change"] == 0:
-                failure = "true"
-                failures_count += 1
-            labels = metrics_config[metric]["labels"]
-            label_string = " ".join(labels) if labels else ""
-            testcase = ET.SubElement(
-                testsuite,
-                "testcase",
-                name=f"{label_string} {metric} regression detection",
-                attrib=run_data,
-            )
-            if failure == "true":
-                failure_element = ET.SubElement(testcase, "failure")
-                failure_element.text = f"{metric} has a value of {value['value']:.2f} with a percentage change of {value['percentage_change']:.2f}% over the previous runs"
+    test_count = 0
+    for metric, value in metrics_config.items():
+        test_count += 1
+        labels = value["labels"]
+        label_string = " ".join(labels) if labels else ""
+        testcase = ET.SubElement(testsuite, "testcase", name=f"{label_string} {metric} regression detection", timestamp=str(int(datetime.now().timestamp())))
+        if [run for run in data_json if not run["metrics"][metric]["percentage_change"] == 0]:
+            failures_count +=1
+            failure = ET.SubElement(testcase,"failure")
```

Comment:
I wonder if we can reduce the output here, to be where the change-point was seen, with 1 value after and 1 before. wdyt? would cleanup the output quite a bit.

Reply:
Yeah, I felt the same that the output was too verbose, we can just have one before and after.

Reply:
yeah, quite honestly, with how we plan to run this in CPT I doubt it will be as verbose. but, just to be careful ;)